common/api/ecschema-editing.api.md

```diff
@@ -1936,12 +1936,17 @@ export class SchemaContextEditor {
     finish(): Promise<SchemaContext>;
     // (undocumented)
     readonly formats: Formats;
-    getSchema(schemaKey: SchemaKey): Promise<MutableSchema | undefined>;
+    // @internal
```

Comment:
What's the motivation for switching `getSchema` and `lookupSchemaItem` to `@internal`?

Reply:
They are not supposed to be called from outside this package